ThreeDSecure/src/main/java/com/braintreepayments/api/threedsecure/ThreeDSecureClient.java

```diff
@@ -256,9 +256,9 @@ public void tokenize(ThreeDSecurePaymentAuthResult paymentAuthResult,
         if (threeDSecureError != null) {
             callbackTokenizeFailure(callback, new ThreeDSecureResult.Failure(threeDSecureError, null));
         } else {
-            ThreeDSecureParams threeDSecureParams = paymentAuthResult.getThreeSecureResult();
```

Comment:
Is there going to be a separate PR for converting the external classes - this one and the `ThreeDSecureRequest` and `Launcher`?)

Reply:
Yep! I'll have a few more follow-up PRs for the remaining 3DS classes.